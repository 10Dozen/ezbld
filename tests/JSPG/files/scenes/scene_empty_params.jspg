# SceneWithEmptyParams |

*goto:
*pre_exec:

Desc line 1.

Desc line 2.

**post_exec:

*