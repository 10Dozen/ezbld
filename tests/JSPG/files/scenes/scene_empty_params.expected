[0]
cls = JSPGScene
name = SceneWithEmptyParams
desc =
    Desc line 1.

    Desc line 2.
